src/util/orders.test.ts

```diff
@@ -77,3 +77,55 @@ describe('buildOrder', () => {
         );
     });
 });
+describe('orderDetails', () => {
+    it('orderDetailsFeeEther should calculate totalFee for buy order ', () => {
+        // given
+        const makerAmount = new BigNumber(5000000000000000000);
+        const takerAmount = new BigNumber(500000000000000000);
+        const orderType = OrderSide.Sell;
+        const MAKER_FEE = '0.1';
+        const TAKER_FEE = '0.05';
+
+        // when
+        const orderInEther = orderDetailsFeeEther(makerAmount, takerAmount, orderType, MAKER_FEE, TAKER_FEE);
+
+        // then
+        const resultExpected = new BigNumber(75000000000000000);
```

Comment:
According to my (manual) calculations, this should also be 0.75. In fact, if you look at the numbers you are using, it makes no sense for this fee to be 10^18 times bigger than the next scenario.

Reply:
Updated